src/client/oicp/CpoOICPClient.ts

```diff
@@ -0,0 +1,1272 @@
+
+    // Build payload
+    const operatorEvseData: OICPOperatorEvseData = {} as OICPOperatorEvseData;
+    operatorEvseData.OperatorID = this.getOperatorID(ServerAction.OICP_PUSH_EVSES);
+    operatorEvseData.OperatorName = this.tenant.name;
+    operatorEvseData.EvseDataRecord = evses;
+
+    const payload: OICPPushEvseDataCpoSend = {} as OICPPushEvseDataCpoSend;
+    payload.ActionType = actionType;
+    payload.OperatorEvseData = operatorEvseData;
+
+    // Log
+    Logging.logDebug({
+      tenantID: this.tenant.id,
+      action: ServerAction.OICP_PUSH_EVSES,
+      message: `Push EVSEs from tenant: ${this.tenant.id}`,
+      module: MODULE_NAME, method: 'pushEvseData',
+      detailedMessages: { payload }
+    });
+
+    // Call Hubject
+    await this.axiosInstance.post(fullUrl, payload
+    ).then(function(response) {
+      console.log('Success! pushEvseData: ', response);
+      pushEvseDataResponse = response.data as OICPAcknowledgment;
+    }).catch((error) => {
+      console.log('Error! pushEvseData: ',error.message);
+    });
+    return pushEvseDataResponse;
+  }
+
+  /**
+   * Push EVSE Status
+   */
+  async pushEvseStatus(evseStatuses: OICPEvseStatusRecord[], actionType: OICPActionType): Promise<OICPAcknowledgment> {
+    let pushEvseStatusResponse: OICPAcknowledgment = {} as OICPAcknowledgment;
+    // Check for input parameter
+    if (!evseStatuses) {
+      throw new BackendError({
+        action: ServerAction.OICP_PUSH_EVSE_STATUSES,
+        message: 'Invalid parameters',
+        module: MODULE_NAME, method: 'pushEvseStatus',
+      });
+    }
+    // Get EVSE Status endpoint url
+    // for QA - environment: https://service-qa.hubject.com/api/oicp/evsepush/v21/operators/{operatorID}/status-records
+    // for PROD - environment: https://service.hubject.com/api/oicp/evsepush/v21/operators/{operatorID}/status-records
+    const fullUrl = this.getEndpointUrl('statuses', ServerAction.OICP_PUSH_EVSE_STATUSES);
+
+    // Build payload
+    const operatorEvseStatus: OICPOperatorEvseStatus = {} as OICPOperatorEvseStatus;
+    operatorEvseStatus.OperatorID = this.getOperatorID(ServerAction.OICP_PUSH_EVSE_STATUSES);
+    operatorEvseStatus.OperatorName = this.tenant.name;
+    operatorEvseStatus.EvseStatusRecord = evseStatuses;
+
+    const payload: OICPPushEvseStatusCpoSend = {} as OICPPushEvseStatusCpoSend;
+    payload.ActionType = actionType;
+    payload.OperatorEvseStatus = operatorEvseStatus;
+
+    // Log
+    Logging.logDebug({
+      tenantID: this.tenant.id,
+      action: ServerAction.OICP_PUSH_EVSE_STATUSES,
+      message: `Push EVSE statuses from tenant: ${this.tenant.id}`,
+      module: MODULE_NAME, method: 'pushEvseStatus',
+      detailedMessages: { payload }
+    });
+
+    // Call Hubject
+    await this.axiosInstance.post(fullUrl, payload
+    ).then(function(response) {
+      console.log('Success! pushEvseStatus: ', response);
```

Comment:
Debug log, use the logging debug helper. Do it everywhere.

Reply:
Yes, I will remove it in the future.
At the moment I need the console logs.